src/olympia/reviewers/models.py

```diff
@@ -600,13 +600,11 @@ def check_has_auto_approval_disabled(cls, version):
     @classmethod
     def check_is_promoted_prereview(cls, version):
         """Check whether the add-on is a promoted addon group that requires
-        pre-review.
-
-        Only applies to listed versions."""
-        if not version.channel == amo.CHANNEL_LISTED:
-            return False
-        promo_group = version.addon.promoted_group(currently_approved=False)
-        return bool(promo_group and promo_group.pre_review)
+        pre-review."""
+        return bool(
```

Comment:
I'm sorry but I cannot spot the difference in the logic between the old code and the new code. It looks like they will both return `False` if `not version.channel == amo.CHANNEL_LISTED`.

Reply:
Yeah, there wasn't any difference in the end - I left it in because it read better to me to have it all in a single statement